portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -30,29 +35,58 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // If user is logged in and has not set a nickname, redirect to nickname page
```

Comment:
nit: Include a period at the end of comments.

Reply:
Done!